src/org/zaproxy/zap/extension/websocket/WebSocketSenderScript.java

```diff
@@ -0,0 +1,7 @@
+package org.zaproxy.zap.extension.websocket;
+
+import javax.script.ScriptException;
+
+public interface WebSocketSenderScript {
+	void onMessageFrame(WebSocketMessage msg) throws ScriptException;
```

Comment:
It could pass the `channelId`/`initiator` too. Those could be provided with the utility object (like in the HTTP Sender, `HttpSenderScriptHelper`, so that we can provide more data/behaviour without changing the interface).